### DISCUSSION TITLE
Question regarding DIB / L0_Cache configuration & eviction policy

### DISCUSSION THREAD

**Discussion by krishnamtibrewala** [ChampSim/ChampSim#413] [Q&A] [answered]
In `champsim_config.json` while configuring DIB what do we mean by `window_size` parameter ? 

Regarding eviction policy of L0_Cache following paper `https://www.cs.virginia.edu/venkat/papers/isca2021a.pdf` suggests micro-op/L0 cache is inclusive of I-Cache & I-TLB. In current implementation of Champsim are we considering this ?

**Comment by ngober** [comment, marked as answer]
In the DIB, the window size is the size of the block. Instructions that fall in the same DIB window will hit the same DIB element.

Currently, we make no effort to maintain inclusive caches. There is an open issue (#350) to address this, and we welcome contributions to this.
